### DISCUSSION TITLE
Help figuring out the right entities to use for circuit drawings

### DISCUSSION THREAD

**Discussion by bbillups-2** [mozman/ezdxf#948] [Q&A]
Hello! 

I am trying to replicate a schematic drawing I made using another module(schemdraw) in ezdxf so it can be a .dxf file. I've used ezdxf for other uses but couldn't quite find an entity to make a circuit and it's associated pins. Will I need to create a custom polyline for this? I saw the circuit drawing featured in this post from 2020 (https://ezdxf.mozman.at/release-v0-13.html) and was wondering if a special entity was used to make the circuit and if I'm just not looking at the documentation closely enough. 

Thanks!

(Or if there is a way to convert a .svg or .jpg to a .dxf drawing in ezdxf, that could also work!)

**Comment by bbillups-2** [comment]
@mozman Hi! Hoping you can help! I've been searching the docs since posting this and still haven't had much luck

**Comment by mozman** [comment]
The example can found here: http://cadkit.blogspot.com (pic_programmer.dxf)

This example is a simple 2D drawing. The DXF format does not support any special features for any design task. You may look at [KiCAD](https://www.multi-circuit-boards.eu/en/support/pcb-data/kicad.html) a free application to create PCB layouts - the producer of this sample DXF file.

**Comment by bbillups-2** [comment]
Thank you very much!
